docs/more-pushmsg.md

```diff
@@ -33,3 +33,53 @@ func main() {
         url.Values{"message": {"My Message"}, "title": {"My Title"}})
 }
 ```
+
+### PHP (using cURL)
+
+```php
+$data = array(
```

Comment:
Is there a reason for using the older array() syntax and not brackets `[]`?

Reply:
Sorry for this it's Updated Now